foreign-chains/solana/sol-prim/src/consts.rs

```diff
@@ -42,3 +42,8 @@ pub const LAMPORTS_PER_SIGNATURE: u64 = 5000u64;
 pub const NONCE_ACCOUNT_LENGTH: u64 = 80u64;
 
 pub const SOL_USDC_DECIMAL: u8 = 6u8;
+
+// todo: confirm this
+pub const MAX_BATCH_SIZE_OF_CONTRACT_SWAP_ACCOUNT_CLOSURES: usize = 10;
+pub const MAX_WAIT_BLOCKS_FOR_SWAP_ACCOUNT_CLOSURE_APICALLS: u32 = 14400; // 1 day
+pub const NONCE_AVAILABILITY_THRESHOLD_FOR_INITIATING_SWAP_ACCOUNT_CLOSURES: usize = 4; // revisit this
```

Comment:
what needs confirming / revisiting? can we do this before this PR is merged?

Reply:
this carried over from the base PR. This is mostly confirmed if @albert-llimos doesnt have any objection to it.

Reply:
sounds good